### PR TITLE
Target Working, Gamestop in Progress

### PR DESCRIPTION
**Pull request by salarsen**
Updated to fix target login issues. Requires users to download GeckoDriver to allow firefox emulation for target in place of Chrome. Target login issue appears to be chrome specific. Added folder select button to point program at geckodriver installation location. Script will auto-stop if a path is not set (probably should set it to check for the driver to exist in said folder). Added setting line for path of driver.

### CODE REVIEW

**Comment by salarsen** [comment]
Also added in task sorting. It will now sort alphabetically based on site to better group tasks for same sites together.

File: sites/target.py [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
+from selenium.webdriver.firefox.options import Options
+
 from utils import random_delay, send_webhook, create_msg
 from utils.selenium_utils import change_driver
-import settings, time
+import settings, time, random
+from test import mypath
 
-options = Options()
+# options = Options()
```
Comment (Strip3s) [inline sites/target.py:16]: I would go ahead and remove the commented code. Seems like firefox is what will need to be used for Target functionality. We can add multi-browser functionality at a higher level across all classes later on if we decide to stick to a pattern at a later date.

**Review by Strip3s — CHANGES_REQUESTED** [review]

File: sites/target.py (hunk 1 above)
Comment (salarsen) [reply sites/target.py:16]: I left it in as I was unsure if firefox supported the same type and implementation of preferences to reduce page load times.

**Comment by salarsen** [comment]
Left in Target chrome options for time being, will remove once I've verified they will not work with firefox. Updated gamestop to remove captcha blocker for more testing. Removed a few print statements that were used for debugging.

**Review by Strip3s — APPROVED** [review]
Sounds good then. Thanks for the contributions recently.

**Comment by salarsen** [comment]
> Sounds good then. Thanks for the contributions recently.

I nabbed a series x halo edition otherwise I'd be keeping these to myself :p

**Pull request merged**